brillig_bytecode/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "brillig_bytecode"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+acir_field.workspace = true
+serde.workspace = true
+num-bigint = "0.4"
+
+[features]
+bn254 = ["acir_field/bn254"]
```

```suggestion
default = ["bn254"]
bn254 = ["acir_field/bn254"]
```

Without a default feature flag, compilation will fail during publishing due to no curve being specified.